.storybook/preview.tsx

```diff
@@ -61,6 +69,9 @@ const preview: Preview = {
         </>
       )
     },
+  },
+  args: {
+    fontSize: '16px'
```

Comment:
Do we want to use `fontSizeBase` from our tokens for this?

Reply:
why this is 16 and in the stories 32? is because this represents the default size and the others are with the double size?

Reply:
yes, exactly. 16 is the default root font size for browsers. The 32 in stories is only used for those stories that are meant to be testing components with 200% zoom.

Reply:
@olliecurtis It's not exactly 100% linked to our Skyscanner typography system. Here the 16px is meant to represent the default browser font size which is a separate thing. Although I don't think either will ever change 😄

Reply:
It's not representative of our base size though, it's representative of the standard browser size which is 16px.
I think all our font sizes are in rem too so technically our base size is not 16px, it's 1rem which with most browser settings ins 16px